yarn.lock

```diff
@@ -4,27 +4,27 @@
 
 "@ampproject/remapping@^2.1.0":
   version "2.2.0"
-  resolved "https://registry.yarnpkg.com/@ampproject/remapping/-/remapping-2.2.0.tgz#56c133824780de3174aed5ab6834f3026790154d"
+  resolved "https://registry.npmjs.org/@ampproject/remapping/-/remapping-2.2.0.tgz"
```

Comment:
Almost done! Last question, I think: are these lockfile changes intentional? Looks like something that might have been done by accident

Reply:
Oof no. I accidentlally did npm install at the beginning. I thought removing package-lock and doing yarn install would essentially reverse what I did…

Reply:
Idk. I tried removing yarn lock, and redoing yarn install, and there’s still a diff…